src/openrct2/ride/TrackDesign.cpp

```diff
@@ -1534,15 +1532,15 @@ static bool track_design_place_ride(TrackDesign* td6, int16_t x, int16_t y, int1
             trackType = TRACK_ELEM_MULTIDIM_INVERTED_90_DEG_UP_TO_FLAT_QUARTER_LOOP;
         }
 
-        track_design_update_max_min_coordinates(x, y, z);
+        track_design_update_max_min_coordinates(_trackPreviewOrigin);
```

Comment:
This is a for loop and `x`, `y` and `z` are changed at the end of the for, whereas `trackPreviewOrigin` are kept constant, so please use the former variables instead:
```suggestion
        track_design_update_max_min_coordinates({ x, y, z });
```